hathor/transaction/transaction_metadata.py

```diff
@@ -335,3 +315,31 @@ def is_in_voided_by(self, item: bytes) -> bool:
         if self.voided_by is None:
             return False
         return item in self.voided_by
+
+    @property
+    def height(self) -> int:
```

Comment:
We have removed access to these attributes all over the code. What do we need these properties for?

Reply:
It was not completely removed from all the code, only a small portion was migrated to the new attributes. Those are kept for compatibility while we finish moving all code. Doing this makes incremental changes easier.

Reply:
Got it. Thanks!